### PR TITLE
Add TV spoken_languages, TV tagline, alternative title type

### LINKED ISSUE #349: tagline is missing from TvShow Object
Firstly thanks very much for a great library - so easy to use.

I can't find the tagline property in the TVShow object. Am I missing something?

Do we just need 

[JsonProperty("tagline")]
public string Tagline { get; set; }

in https://github.com/LordMike/TMDbLib/blob/master/TMDbLib/Objects/TvShows/TvShow.cs

based on https://developers.themoviedb.org/3/tv/get-tv-details

### PR DESCRIPTION
**Pull request by wwarby**
Adds a couple of missing fields and corrects alphabetisation of a couple of missing properties

Fixes #349, #351

### CODE REVIEW

File: TMDbLib/Objects/General/AlternativeTitle.cs [hunk 1]
```diff
@@ -12,5 +12,11 @@ public class AlternativeTitle
 
         [JsonProperty("title")]
         public string Title { get; set; }
+        
+        /// <summary>
+        /// The type of title (e.g. working title, DVD title, modern title)
+        /// </summary>
+        [JsonProperty("type")]
+        public string Type { get; set; }
```
Comment (LordMike) [inline TMDbLib/Objects/General/AlternativeTitle.cs:20]: Can we make this an enum?

Comment (wwarby) [reply TMDbLib/Objects/General/AlternativeTitle.cs:20]: No I don’t think you can - on the TMDB website this is actually editable as a free text field, not a drop down - there are no fixed values for the property unfortunately.

**Comment by wwarby** [comment]
Just wondering if you might be able to do an alpha release to Nuget soon containing this PR? I’d really like to start using the data it make available :)

**Comment by LordMike** [comment]
Done. :)

**Pull request merged**